api/tests/unit/domain/usecases/find-user-for-oidc-reconciliation_test.js

```diff
@@ -154,153 +146,33 @@ describe('Unit | UseCase | find-user-for-oidc-reconciliation', function () {
     });
   });
 
-  context('when user has an oidc authentication method', function () {
-    context('when externalIdentifier and externalIdentityId are different', function () {
-      it('should throw an DifferentExternalIdentifierError', async function () {
-        // given
-        const oidcAuthenticationMethod = domainBuilder.buildAuthenticationMethod.withPoleEmploiAsIdentityProvider({
-          externalIdentifier: '789fge',
-        });
-        pixAuthenticationService.getUserByUsernameAndPassword.resolves({ id: 2 });
-        authenticationMethodRepository.findByUserId.resolves([oidcAuthenticationMethod]);
-        authenticationSessionService.getByKey.resolves({
-          sessionContent: {},
-          userInfo: { externalIdentityId: '123abc' },
-        });
-
-        // when
-        const error = await catchErr(findUserForOidcReconciliation)({
-          authenticationKey: 'authenticationKey',
-          email: 'ane.trotro@example.net',
-          password: 'pix123',
-          identityProvider: 'POLE_EMPLOI',
-          authenticationSessionService,
-          pixAuthenticationService,
-          authenticationMethodRepository,
-          userRepository,
-        });
-
-        // then
-        expect(error).to.be.instanceOf(DifferentExternalIdentifierError);
+  context('when user has an oidc authentication method and external identifiers are different', function () {
```

Comment:
Suggestion 
Réorganiser les tests en ajoutant du contexte pour mieux comprendre les cas possible

-  `context('when user has an oidc authentication method') 
   - context('external identifiers are the same'
       - it('should find pix user and their oidc authentication method',
       -  it('should retrieve user session content',
   - context('external identifiers are different'
       - it('should throw an DifferentExternalIdentifierError'

Reply:
Doublon. J'ai perdu ma connexion internet à un moment et je voyais plus mes commentaires, je resolve